functions/src/services/__tests__/firestoreDailySession.test.js

```
@@ -0,0 +1,303 @@
+      await firestoreDailySessionRepo.save(testDate, testPatients, testUid);
+
+      expect(mockDb.settings).toHaveBeenCalledWith({
+        host: 'localhost:8080',
+        ssl: false
+      });
+
+      // Restore original environment
+      if (originalHost) {
+        process.env.FIRESTORE_EMULATOR_HOST = originalHost;
+      } else {
+        delete process.env.FIRESTORE_EMULATOR_HOST;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)